shortcodes/pmpro_account.php

```diff
@@ -153,25 +153,12 @@ function pmpro_shortcode_account($atts, $content=null, $code="")
 								</td>
 								<td class="<?php echo esc_attr( pmpro_get_element_class( 'pmpro_account-membership-expiration' ) ); ?>">
 									<?php
-										$expiration_text = '<p>';
 										if ( $level->enddate ) {
-											$expiration_text .= date_i18n( get_option( 'date_format' ), $level->enddate );
-											/**
-											 * Filter to include the expiration time with expiration date
-											 *
-											 * @param bool $pmpro_show_time_on_expiration_date Show the expiration time with expiration date (default: false).
-											 *
-											 * @return bool $pmpro_show_time_on_expiration_date Whether to show the expiration time with expiration date.
-											 *
-											 */
-											if ( apply_filters( 'pmpro_show_time_on_expiration_date', false ) ) {
-												$expiration_text .= ' ' . date_i18n( get_option( 'time_format', __( 'g:i a' ) ), $level->enddate );
-											}
+											$expiration_text = pmpro_get_membership_expiration_text( $level->enddate, $level, $current_user );
 										} else {
-											$expiration_text .= esc_html_x( '&#8212;', 'A dash is shown when there is no expiration date.', 'paid-memberships-pro' );
+											$expiration_text = pmpro_get_membership_expiration_text( esc_html_x( '&#8212;', 'A dash is shown when there is no expiration date.', 'paid-memberships-pro' ), $level, $current_user );											

```

Comment:
The doc block for `pmpro_get_membership_expiration_text()` currently says that the first parameter must be an int. A string is passed here.

Could we make an assumption that "Never" is shown for an empty expiration date in the Dashboard and `&#8212;` is shown on the frontend? If so, we can abstract this logic directly into the `pmpro_get_membership_expiration_text()` function when an empty enddate is passed.